### DISCUSSION TITLE
Emoticons

### DISCUSSION THREAD

**Discussion by StuartMcD** [solspace/craft-freeform#733] [Feature Requests]
Entering an emoticon in a textarea field in a freeform and it's saved to DB as (don't know tech term) plaintext `:fist:` not `&#x1f5f3;️`

Entering the same emoticon into a craft textarea or title saves as code `&#x1f5f3;️`

Viewing the emoticon inside a Freeform entry via `craft.freeform.submissions` front end - rendered literal (plaintext)
Viewing the emoticon inside admin email - rendered graphically

TL:DR
Currently using FF as a comment section works great but emoticons dont render - is this Craft, FF or web-standards & can it be addressed ?

**Comment by kjmartens** [comment]
Hi @StuartMcD,

As far as I can tell on my end, both Freeform and Craft entries seem to store emojis like 😬 as `:grimacing_face:` in the database. However, I can see that when attempting to render inside the CP or front-end templates, it doesn't convert in Freeform like it does for Craft.

Freeform needs to convert the `:grimacing_face:` back to Unicode in order for it to work the same way. It's not something super quick, but we'll likely add support for it in the not-too-distant future. 🙂

**Reply by kjmartens** [reply]
@StuartMcD I've updated my response here. We've added this to the feature requests list and will likely add support for it within a couple of months as we juggle priorities with the upcoming Freeform 5. 🙂
